software/firmware/README.md

```diff
@@ -1,6 +1,6 @@
 # Firmware
 
 Please download the most recent MicroPython firmware from [the MicroPython website](https://micropython.org/download/rp2-pico/).
-  
-The 'flashnuke.uf2' file is a last resort, and will wipe all data stored on your Pico, including calibration data and custom programs.  
-It is only to be used when all other troubleshooting options have been covered, as found in [troubleshooting.md](https://github.com/Allen-Synthesis/EuroPi/blob/main/troubleshooting.md)
+
+The 'flashnuke.uf2' file is a last resort, and will wipe all data stored on your Pico, including calibration data and custom programs.
```

Comment:
Optional: I think we should remove `flashnuke.uf2` from the repo and instead provide a URL to the official source of the file, and update any references in our docs.

Reply:
I'm going to open that as a separate issue.

Reply:
https://github.com/Allen-Synthesis/EuroPi/issues/59